test/tla/cli-integration-tests.md

```diff
@@ -1293,3 +1293,25 @@ Typing input error: Expected a type annotation for VARIABLE x
 EXITCODE: ERROR (99)
 ```
 
+## Running the config command
```

Comment:
Should we should check the case when the directory needs to be created, and when it cannot be created? We should be able to do this relatively easily in the CI by overriding `user.home` I think? For the case when the dir cannot be created, perhaps we could just configure it be created at a non-existent path?

Reply:
I guess, this is happening already, as the default image does not have the directory `$HOME/.tlaplus`. We just did not have a test for `config`, and this happened to produce a bug :)